packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -128,14 +116,19 @@ export default class SaytDriverPlugin implements Plugin {
    *
    * @param event Event that contains the Search API request payload.
    */
-  fetchProductData(event: CustomEvent<SearchRequestConfig>) {
+  fetchProductData(event: CustomEvent<SaytProductsRequestPayload>) {
     const { query, group, config } = event.detail;
     this.sendSearchApiRequest(query, config)
       .then(results => {
-        this.core[this.eventsPluginName].dispatchEvent(this.productResponseEvent, { results, group });
+        const payload: SaytProductsResponsePayload = {
+          ...results,
```

Comment:
At some point in the lifetime of this ticket, the intermediate `results` property was removed and `products` was promoted (or perhaps `results` was added in a later commit and not properly picked up in the rebase). If this was against the design of this particular payload, we can add `results` back in, but I think it's consistent with our design goals, considering that the primary data coming back from this payload are the products.

It would be a breaking change had we released the package prior to this PR, but because we have not yet done so, we should be okay as long as the View layer is updated accordingly.